carma_wm_ctrl/src/WMBroadcaster.cpp

```diff
@@ -245,6 +259,14 @@ void WMBroadcaster::addRegionAccessRule(std::shared_ptr<Geofence> gf_ptr, const
   gf_ptr->regulatory_element_ = regulatory_element;
 }
 
+void WMBroadcaster::addRegionMinimumGap(std::shared_ptr<Geofence> gf_ptr, double min_gap, const std::vector<lanelet::Lanelet>& affected_llts, const std::vector<lanelet::Area>& affected_areas) const
+{
+  auto regulatory_element = std::make_shared<lanelet::DigitalMinimumGap>(lanelet::DigitalMinimumGap::buildData(lanelet::utils::getId(), 
+                                        min_gap, affected_llts, affected_areas, { lanelet::Participants::VehicleCar }));
```

Comment:
The participant list you have here should be based on the TrafficControlParam.vclasses list. You can use the  participantsChecker() method to extract that from the control message. There is also a invertParticipants() method if that is needed.

Reply:
https://github.com/usdot-fhwa-stol/carma-platform/pull/1197/commits/761f745a65683501b99f5387e17f2388de375e83